docs/about.md

```diff
@@ -15,7 +15,7 @@ The **backup** operation:
 1. Calls the cloud provider API to make disk snapshots of persistent volumes, if specified.
 
 You can optionally specify hooks to be executed during the backup. For example, you might
-need to tell a database to flush its in-memory buffers to disk before taking a snapshot. [More about hooks][10].
+need to tell a database to flush its in-memory buffers to disk before taking a snapshot.
```

Comment:
Can you leave this link in place, and add in a [10] at the bottom that points to `docs/hooks.md`?

Reply:
sure

Reply:
Done. I had to use `push --force` to `amend` the previous commit.
Was there a way to do it differently or I had to use `--force` since the commit was already pushed?

Reply:
Nope, that's the right way to do it if you're amending an existing commit

Reply:
Okay, thanks!

So if this PR gets merged is there some kind of form to fill out for Hacktoberfest swag? 😄